README.md

```diff
@@ -15,6 +15,7 @@
 - `PAGERDUTY_READ_ONLY_TOKEN` is used for reading schedules and checking the overlaps.
 - `SCHEDULES` array can contain one or more `SCHEDULE` items to check
 - every `SCHEDULE` should have a `NOTIFICATIONS` to create incident or send message if overlap is found
+- `SCHEDULE` can contain a `DAYS` key, which speficies numbers of days in UTC (0 = Sunday) for verification (`[0, 1, 2, 3, 4]` below represents days without weekend)
```

Comment:
Since we don't check particular time yet, Friday is omitted because of weekend on-call. Sunday is there because normal shift starts there. Next change will add some time checks as well, so I hope it will clarify it a bit.